src/System.Text.Json/tests/Serialization/Value.ReadTests.ConcurrentCollections.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Concurrent;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class ValueTests
+    {
+        [Fact]
+        public static void Read_ConcurrentCollection()
+        {
+            ConcurrentDictionary<string, string> cd = JsonSerializer.Deserialize<ConcurrentDictionary<string, string>>(@"{""key"":""value""}");
```

Comment:
IMO having a single test for each of these doing both deserialize + serialize as well as verifying rount-tripping capability (e.g. an extra serialize or deserialize) would be more readable and better test the functionality.

I realize some existing tests separate the "read" and "write", but going forward I don't think that is better.